metricbeat/module/kubernetes/state_storageclass/state_storageclass.go

```diff
@@ -66,10 +70,12 @@ func NewStorageClassMetricSet(base mb.BaseMetricSet) (mb.MetricSet, error) {
 				"kube_storageclass_created": p.Metric("created", p.OpUnixTimestampValue()),
 			},
 			Labels: map[string]p.LabelMap{
-				"storageclass":      p.KeyLabel("name"),
-				"provisioner":       p.Label("provisioner"),
-				"reclaimPolicy":     p.Label("reclaim_policy"),
-				"volumeBindingMode": p.Label("volume_binding_mode"),
+				"storageclass":        p.KeyLabel("name"),
+				"provisioner":         p.Label("provisioner"),
+				"reclaimPolicy":       p.Label("reclaim_policy"),
+				"reclaim_policy":      p.Label("reclaim_policy"),
+				"volumeBindingMode":   p.Label("volume_binding_mode"),
+				"volume_binding_mode": p.Label("volume_binding_mode"),
```

Comment:
Why this here? Is it different than volumeBindingMode?

Reply:
yes, it was changed in kube-state-metrics v2 - https://github.com/kubernetes/kube-state-metrics/pull/1107/files and `reclaimPolicy` was renamed to `reclaim_policy`, and `volumeBindingMode` to `volume_binding_mode`.
locally when I was testing orchestrator field I was getting:

to support both versions of kube-state-metrics I've added this to mapping (< 2.0 - used `reclaimPolicy` and  `volumeBindingMode`, example - https://github.com/elastic/beats/blob/main/metricbeat/module/kubernetes/_meta/test/ksm.v1.8.0#L1370, versions > 2.0 - https://github.com/elastic/beats/blob/main/metricbeat/module/kubernetes/_meta/test/ksm.v2.0.0#L1545)